sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -539,22 +540,17 @@ def writer(self, test_bigquery_client=None, buffer_size=None):
 
 
 class BigQueryWriteFn(DoFn):
-  """A ``DoFn`` that streams writes to BigQuery once the table is created.
-  """
+  """A ``DoFn`` that streams writes to BigQuery once the table is created."""
+
+  MAX_BUFFERED_ROWS = 2000
+  MAX_BUFFER_SIZE = 500
```

Comment:
`MAX_BATCH_SIZE` for matching `self._max_batch_size`?

Reply:
Thanks. Done.